lib/pbench/test/unit/server/conftest.py

```diff
@@ -991,20 +1020,9 @@ def tarball(tmp_path):
     """
     filename = "pbench-user-benchmark_some + config_2021.05.01T12.42.42.tar.xz"
     datafile = tmp_path / filename
-    metadata = MetadataLog()
-    metadata.add_section("pbench")
-    metadata.set("pbench", "date", "2002-05-16")
-    metadata_file = tmp_path / "metadata.log"
-    with metadata_file.open("w") as meta_fp:
-        metadata.write(meta_fp)
-    with tarfile.open(datafile, "w:xz") as tar:
-        tar.add(str(metadata_file), arcname=f"{Dataset.stem(filename)}/metadata.log")
-    md5 = hashlib.md5()
-    md5.update(datafile.read_bytes())
-    md5file = tmp_path / (filename + ".md5")
-    md5file.write_text(md5.hexdigest())
+    md5file, md5hash = make_tarball(datafile, "2002-05-16")
 
-    yield datafile, md5file, md5.hexdigest()
+    yield datafile, md5file, md5hash
 
     # Clean up after the test case
```

Comment:
Should we be unlinking the `metadata.log` file, either here or in `make_tarball()`?

Reply:
Probably, although that's not a new problem and I didn't even think about it.

Reply:
My mistake...I thought you had added the `metadata.log` file creation, although clearly you hadn't.